src/Microsoft.Management.Deployment.CsWinRTProjection/Microsoft.Management.Deployment.CsWinRTProjection.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-windows10.0.22000.0;net8.0-windows10.0.22000.0</TargetFrameworks>

```

Comment:
On second thought, I think we should do .NET 8 only.
.NET 6 is literally going EOL next Tuesday 😄